sitemap/fetcher_test.go

```diff
@@ -196,8 +202,8 @@ func TestFetcher(t *testing.T) {
 			So(receivedScrollID, ShouldEqual, "scroll_id_1")
 		})
 		Convey("Temporary sitemap file should be created and then cleaned up", func() {
-			So(filename[0], ShouldContainSubstring, "sitemap")
-			_, err := os.Stat(filename[0])
+			So(filename[config.English], ShouldContainSubstring, "sitemap")
+			_, err := os.Stat(filename[config.English])
```

Comment:
The err variable needs to be initialised to avoid a potential nil pointer in the next line.